recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set ucx_version = "1.6.1" %}
```

Comment:
Should we update the ucx_version as well ?

Reply:
There hasn't been a new stable release. So no.

Reply:
Oh, this is going into master, not the RC label

Reply:
We can also move it there too. Though this was mostly cleanup. Had some other plans for changes here that I haven't gotten to yet.